packages/service_authentication/src/services/jwt.service.ts

```
@@ -0,0 +1,40 @@
+import { AuthUser } from "@fakelook/common/src/models/interfaces";
+import { sign, verify } from "jsonwebtoken";
+import { appLoggerService } from ".";
+import { general } from "../constants";
+
+export const createAccessToken = (refreshToken: string): string | undefined => {
+    const valid = verifyToken(refreshToken);
+    if (valid) {
+        appLoggerService.verbose("generate an access JWT token:");
+        return sign(refreshToken, general.jwtSecret, { expiresIn: "15m" });
+    } else {
+        appLoggerService.verbose("failed to decode refresh token");
+        return undefined;
```

Why not `throw` if the refresh token was invalid?